### DISCUSSION TITLE
where is vue-vine vsc plugin?

### DISCUSSION THREAD

**Discussion by moushicheng** [vue-vine/vue-vine#58] [Q&A] [answered]
this plugin is so helpful but i can't found this plugin in doc or plugin market, plz tell me where it is ,thx🙏

**Comment by baiwusanyu-c** [comment, marked as answer]
vine has not released any version, because it is waiting for volar2.0 to support and improve vine’s language tools

**Comment by so1ve** [comment]
This project is still under heavy development and it hasn't released any releases. The vscode extension is not published too because Volar.js still lacks some important functionality that vue-vine depends on.
